spec/factories/form_objects/email_confirmation_form.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :email_confirmation_form, class: "EmailConfirmationForm" do
+    send_confirmation { nil }
+    confirmation_email_address { nil }
+    notify_reference { Faker::Internet.uuid }
```

Comment:
```suggestion
    send_confirmation { nil }
    confirmation_email_address { nil }
    notify_reference {  SecureRandom.uuid }
    
    factory :send_email_confirmation_form do
    	send_confirmation { true }
      confirmation_email_address { Faker::Internet.email }
    end
```

It feels better to have a "different" factory for instances where the user would like to send emails and its feels better than using a trait to declare these values

You should be able to do something like 
`build :send_email_confirmation_form` 

Could you maybe make use of this factory in your specs?

Reply:
Good suggestion - have done this (changed the name slightly)

Reply:
Like you the name you used, its more descriptive 👍